tests/tools/test_florence2_sam2.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import pytest
+from PIL import Image
+
+from vision_agent_tools.tools.florence2_sam2 import Florence2SAM2
+
+
+def test_successful_florence2_sam2_image():
+    """
+    This test verifies that CLIPMediaSim returns a valid iresponse when passed a target_text
```

Comment:
I think you should change `CLIPMediaSim` to `Florence2SAM2`

Reply:
done. thanks.